prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    const fullName = 'Joseph Park';
+    console.log('value of fullName:', fullName);
+
+    const numberOfPets = 0;
+    console.log('value of numberofPets:', numberOfPets);
+
+    const likesVegetables = (true);
+    console.log('value of likesVegetables:', likesVegetables);
+
+    const nothing = 'null';
```

Comment:
We want to leave the quotes off of `null` so that the JavaScript engine doesn't interpret it as a string.

Reply:
Got it!